app/src/common/shared/com/igalia/wolvic/browser/engine/EngineProvider.kt

```diff
@@ -43,6 +43,7 @@ object EngineProvider {
             builder.screenSizeOverride(settingsStore.maxWindowWidth, settingsStore.maxWindowHeight)
             builder.inputAutoZoomEnabled(false)
             builder.doubleTapZoomingEnabled(false)
+            builder.forceUserScalableEnabled(false)
```

Comment:
Why this? If not strictly needed I'd very much prefer not to change it, because it might potentially cause regression. Or is it safe to disable it? What's exactly the effect?

Reply:
Talked offline. This setting disables force zooming when the content (web page) disables it, so we should be safe.